### PR TITLE
Remove color from horizontal lines

### LINKED ISSUE #47: Color of horizontal Lines incompatible with dark Mode
On a web page with white text and a dark background ("dark mode"), the pseudocode block looks a bit strange.

In particular, the horizontal lines remain black, even though the text and the vertical lines are  white.

pseudocode_bug-black_lines

To fix this, I suggest replacing 
```css
border: 2px solid black;
```
inside of `pseudocode.css` with:
```css
border-width: 2px;
border-style: solid;
```
This way you could use pseudocode.js also on a dark website.

### PR DESCRIPTION
**Pull request by Mavoort**
I hereby request that the colour of the horizontal lines be removed,
so that the pseudocode diagram looks the same on a website with a dark background.

Fixes #47 .

### CODE REVIEW

File: static/pseudocode.css [hunk 1]
```diff
@@ -9,11 +9,14 @@
 .ps-root .ps-algorithm {
     margin: 0.8em 0;
     /* algorithm environment has borders; algorithmic has not */
-    border-top: 3px solid black;
-    border-bottom: 2px solid black;
+    border-top-width: 3px;
+    border-top-style: solid;
+    border-bottom-width: 2px;
+    border-bottom-style: solid;
```
Comment (SaswatPadhi) [inline static/pseudocode.css]:
```suggestion
    border-top: 3px solid;
    border-bottom: 2px solid;
```

File: static/pseudocode.css [hunk 2]
```diff
@@ -9,11 +9,14 @@
...
 }
 .ps-root .ps-algorithm.with-caption > .ps-line:first-child {
-    border-bottom: 2px solid black;
+    border-bottom-width: 2px;
+    border-bottom-style: solid;
```
Comment (SaswatPadhi) [inline static/pseudocode.css]:
```suggestion
    border-bottom: 2px solid;
```

**Review by SaswatPadhi — CHANGES_REQUESTED** [review]
Hi @Mavoort,

Thanks for this fix! LGTM, but I suggest simplifying the style.

**Comment by Mavoort** [comment]
Thanks for the suggestion! I'm not an expert at CSS, so I didn't know that. It should be good now.

**Review by SaswatPadhi — APPROVED** [review]

**Pull request merged**